src/Gir/Marshal/SymbolTable.Builtin.cs

```diff
@@ -12,11 +12,12 @@ void RegisterBuiltIn (bool nativeWin64)
 			AddType (new StringMarshal ("gfilename", "string", "", "IntPtr"));//, "GLib.Marshaller.StringToFilenamePtr({0})", "GLib.Marshaller.FilenamePtrToStringGFree({0})"));
 			AddType (new StringMarshal ("gchar", "string", "", "IntPtr"));//, "GLib.Marshaller.StringToPtrGStrdup({0})", "GLib.Marshaller.PtrToStringGFree({0})"));
 			AddType (new StringMarshal ("char", "string", "", "IntPtr"));//, "GLib.Marshaller.StringToPtrGStrdup({0})", "GLib.Marshaller.PtrToStringGFree({0})"));
+			AddType (new StringMarshal ("utf8", "string", "", "IntPtr"));//, "GLib.Marshaller.StringToPtrGStrdup({0})", "GLib.Marshaller.PtrToStringGFree({0})"));
```

Comment:
Not sure if we need the others. It seems like the string type name is utf8 for gchar and char in gir.

Reply:
I added `char` because I ran into it. The others I grabbed because of the old generator.

Reply:
Yep, if we go down this route, utf8 is the only one we need.